SQL/0000-00-00-schema.sql

```diff
@@ -1287,6 +1288,21 @@ CREATE TABLE `user_login_history` (
   PRIMARY KEY (`loginhistoryID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+-- ********************************
+-- user password_recovery tables
+-- ********************************
+
+CREATE TABLE `password_recovery` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `ip` int(10) NOT NULL,
+  `user_id` int(10) unsigned NOT NULL,
+  `email` varchar(100) NOT NULL,
+  `token` varchar(65) NOT NULL,
+  `time` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`ID`),
+  CONSTRAINT `fk_password_recovery_users_1` FOREIGN KEY (`email`) REFERENCES `users` (`Email`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
I think this table adds yet another new naming/capitalization scheme to the schema? We really need to come up with one official way..

But anyways, there's also a foreign key missing on the `user_id` column and I'm not sure why the `time` column has the `on update` trigger. What's it supposed to be the time of?

Reply:
since Alex, every new PR i've been making or reviewing I'm expecting snake case...

unless I am modifying a table already followin another convention. 

So no this follows the same convention i've been using and that I assumed was the convention for new structures

Reply:
I don't think that's ever been the case.

Reply:
lol then add it to the list